### PR TITLE
[do not merge]

### PR DESCRIPTION
**Pull request by NimJay**
* I have created this pull-request for the purpose of:
    * generating a static staging URL for some experimental changes
    * receiving feedback on Spanish translations

### CODE REVIEW

File: src/productcatalogservice/products.json
Comment (NimJay) [inline src/productcatalogservice/products.json:1]: I have temporarily hard-coded Spanish product name/description translations inside `src/productcatalogservice/products.json`.

File: src/productcatalogservice/products.json [hunk 1]
```diff
@@ -2,8 +2,8 @@
     "products": [
         {
             "id": "OLJCESPC7Z",
-            "name": "Sunglasses",
-            "description": "Add a modern touch to your outfits with these sleek aviator sunglasses.",
+            "name": "Gafas de sol",
+            "description": "Dale un toque moderno a tus outfits con estas elegantes gafas de sol aviador.",
```
Comment (cortespao) [inline src/productcatalogservice/products.json:6]:
```suggestion
            "description": "Dale un toque moderno a tus outfits con estas elegantes gafas de sol estilo aviador.",
```

File: src/productcatalogservice/products.json [hunk 2]
```diff
@@ -14,8 +14,8 @@
         },
         {
             "id": "66VCHSJNUP",
-            "name": "Tank Top",
-            "description": "Perfectly cropped cotton tank, with a scooped neckline.",
+            "name": "Top de tirantes",
+            "description": "Top de algodón de corte perfecto, con escote en pico.",
```
Comment (cortespao) [inline src/productcatalogservice/products.json:18]:
```suggestion
            "description": "Top de algodón de corte perfecto, con escote redondo.",
```

File: src/productcatalogservice/products.json [hunk 3]
```diff
@@ -62,8 +62,8 @@
         },
         {
             "id": "0PUK6V6EV0",
-            "name": "Candle Holder",
-            "description": "This small but intricate candle holder is an excellent gift.",
+            "name": "Soporte para velas",
+            "description": "Este pequeño pero intrincado portavelas es un excelente regalo.",
```
Comment (cortespao) [inline src/productcatalogservice/products.json:66]:
```suggestion
            "description": "Este pequeño pero lleno de detalles portavelas es un excelente regalo.",
```

File: src/productcatalogservice/products.json [hunk 4]
```diff
@@ -98,8 +98,8 @@
         },
         {
             "id": "6E92ZMYYFZ",
-            "name": "Mug",
-            "description": "A simple mug with a mustard interior.",
+            "name": "Taza",
+            "description": "Una taza simple con interior mostaza",
```
Comment (cortespao) [inline src/productcatalogservice/products.json:102]:
```suggestion
            "description": "Una taza simple con interior color mostaza",
```

**Review by cortespao — COMMENTED** [review]
Just some nits on the translation

File: src/frontend/handlers.go [hunk 5]
```diff
@@ -26,6 +26,7 @@ import (
 	"strings"
 	"time"
 
+	"github.com/GoogleCloudPlatform/microservices-demo/src/frontend/i18n"
```
Comment (NimJay) [inline src/frontend/handlers.go:29]: Thanks for separating this into its own module (`i18n`). Looks clean!

**Comment by NimJay** [comment]
Closing this since Cloud Next 23 is now done.

**Pull request closed without merging**